lib/shipit.rb

```diff
@@ -206,6 +206,11 @@ def default_inactivity_timeout
     secrets.commands_inactivity_timeout || 5.minutes.to_i
   end
 
+  def automatically_prepend_bundle_exec
+    # Prepending `bundle exec` automatically is deprecated
+    Rails.env.production? && ENV.fetch('SHIPIT_PREPEND_BUNDLE_EXEC', '1') != '0'
```

Comment:
Also we should probably enable the new behavior on new apps via the template.rb: https://github.com/Shopify/shipit-engine/blob/96e98e71742a85d47bf529e21b63b9e682147ef3/template.rb

And finally we should probably log an `ActiveSupport::Deprecation.warn` on boot if that env var isn't set.